conda_forge_tick/feedstock_parser.py

```diff
@@ -285,8 +291,17 @@ def populate_feedstock_attributes(
     req = _get_requirements(yaml_dict)
     sub_graph["req"] = req
 
+    # set name and version
     keys = [("package", "name"), ("package", "version")]
     missing_keys = [k[1] for k in keys if k[1] not in yaml_dict.get(k[0], {})]
+    for k in keys:
+        if k[1] not in missing_keys:
+            sub_graph[k[1]] = yaml_dict[k[0]][k[1]]
+
+    # set the url and hash
```

Comment:
Where are we getting these pieces of information now, or are they no longer parts of the metadata?

Reply:
They come from the metadata in the rendered recipe.

Reply:
Does the `hash_type` key still exist at top level?

Reply:
yes